crates/primitives/src/specification.rs

```diff
@@ -128,3 +133,30 @@ spec!(LATEST, LatestSpec);
 spec!(BEDROCK, BedrockSpec);
 #[cfg(feature = "optimism")]
 spec!(REGOLITH, RegolithSpec);
+
+#[cfg(test)]
+mod tests {
+    use super::*;
+
+    #[cfg(feature = "optimism")]
```

Comment:
nit: put the feature flag above the `#[cfg(test)]` directive rather than over each individual test.

Reply:
Was going to optimistically add non-optimism tests, but figured i'd just upstream them